tests/phpunit/tests/widgets.php

```diff
@@ -686,6 +686,53 @@ public function test_wp_widget_get_settings() {
 		$this->assertArrayNotHasKey( 0, $never_used );
 	}
 
+	/**
+	 * @covers WP_Widget::get_settings
+	 * @ticket 54677
```

Comment:
Per [56070](https://core.trac.wordpress.org/ticket/56070), `@ticket` should precede `@covers` and there should be a line break between them.

i.e.
```php
/**
 * @ticket 54677
 *
 * @covers WP_Widget::get_settings
 */
```
This also applies to the other test method.